app/javascript/src/Policies/components/CustomPolicy.jsx

```diff
@@ -117,24 +62,35 @@ function CustomPolicyForm ({policy, onChange}: {policy: Policy, onChange: OnChan
   )
 }
 
-function Form ({initialPolicy}: {initialPolicy: Policy}): React.Node {
+function CustomPolicyEditor ({initialPolicy}: {initialPolicy: Policy}): React.Node {
```

Comment:
The return type `React.Node` is not mandatory. In some files we are using it, in others we aren't. We should agree in using it or not. At a glance, most files don't have it, so we could follow that way, unless you have a reason to explicitly include it.

Reply:
ok, then less is better